.github/workflows/release.yml

```diff
@@ -70,12 +48,12 @@ jobs:
           gpg_passphrase: ${{ secrets.BUILD_PASSPHRASE }}
           nexus_username: ${{ secrets.SONATYPE_USERNAME }} # These are placeholders; we're publishing a Docker image
           nexus_password: ${{ secrets.SONATYPE_PASSWORD }}
-          maven_profiles: release,${{ env.NIGHTLY_BUILD }}
           maven_args: >
-            -Drevision=${{ env.RELEASE_TAG }}
+            -Drevision=${{ github.event.release.tag_name }}
             -ntp -Dorg.slf4j.simpleLogger.log.net.sourceforge.pmd=error
             -D${{ matrix.build_property }}=${{ secrets.KAKADU_VERSION }}
             -DskipNexusStagingDeployMojo=${{ env.SKIP_JAR_DEPLOYMENT }}
```

Comment:
So this could be hard-coded to `true` and the ENV removed.

Reply:
Why is that? I thought the value of `revision` is the version identifier

Reply:
Ah ok, gotcha. They could at least highlight the line in yellow or something 😒